docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -9,27 +9,33 @@
 
 You need a connected environment to install the {PluginName}.
 
-The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2022-03 and Red Hat CodeReady Studio 12.21.3.GA.
+The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2023-06.
```

Comment:
Version 2023-06 has an issue , Reports do not open 
https://issues.redhat.com/browse/WINDUP-4043.

Please document 2023-03 as the version tested.